gcovr/tests/test_args.py

```diff
@@ -29,6 +29,12 @@ def test_version(capsys):
     assert c.exception.code == 0
 
 
+def test_incompatible_gcov(capsys):
+    c = capture(capsys, ['--gcov-executable=/bin/sh'])
+    assert c.exception.code != 0
+    assert c.err.startswith("(ERROR) gcov command: '/bin/sh' does not support ")
```

Comment:
good test, but /bin/sh doesn't exist on Windows systems. To make this test cross-platform, a phony gcov script should be created in the gcovr/tests directory. All it has to do is output a help message mentioning some but not all of the required flags.

Reply:
What about:
```
c = capture(capsys, ['--gcov-executable="' + sys.executable +" --help"'])
```

Reply:
That looks good – except of course that `--help` will be added by the check function, so we don't have to specify it here.